src/main/java/com/salesforce/storm/spout/dynamic/kafka/Consumer.java

```diff
@@ -330,6 +364,48 @@ public Record nextRecord() {
     }
 
     /**
+     * Reports kafka consumer statistics at an interval.
+     */
+    private void reportStatus() {
+        // If we've reported status more recently than 30 seconds we skip over this.
+        if (lastReportedStatusMillis != null && TimeUnit.MILLISECONDS.toSeconds(clock.millis() - lastReportedStatusMillis) < 30) {
```

Comment:
So we have a hardcoded value of 60 in StormRecorder, and a hardcoded value of 30 here.  I think for now that's fine, in the future we may want to think about making them configurable somewhere.